src/msix/unpack/InflateStream.cpp

```diff
@@ -142,6 +142,8 @@ namespace MSIX {
 
     InflateStream::~InflateStream()
     {
+        m_compressedBuffer = nullptr;

```

Comment:
It seems odd that there is memory pressure from having this allocated while `Cleanup` is running.